### PR TITLE
further post repo move cleanup

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: website/docs/d/cosmosdb_account.html.markdown [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following attributes are exported:
 
 * `virtual_network_rule` - Subnets that are allowed to access this CosmosDB account.
 
-* `enable_multiple_write_locations` - If multi-master is enabled for this Cosmos DB account.
+* `enable_multiple_write_locations` - If multi-main is enabled for this Cosmos DB account.
```
Comment (WodansSon) [inline website/docs/d/cosmosdb_account.html.markdown]:
```suggestion
* `enable_multiple_write_locations` - Are multiple write locations enabled for this Cosmos DB account?
```

File: website/docs/d/cosmosdb_account.html.markdown [hunk 2]
```diff
@@ -86,13 +86,13 @@ The following attributes are exported:
 
 * `write_endpoints` - A list of write endpoints available for this CosmosDB account.
 
-* `primary_key` - The Primary master key for the CosmosDB Account.
+* `primary_key` - The Primary key for the CosmosDB Account.
 
-* `secondary_key` - The Secondary master key for the CosmosDB Account.
+* `secondary_key` - The Secondary main key for the CosmosDB Account.
```
Comment (WodansSon) [inline website/docs/d/cosmosdb_account.html.markdown]:
```suggestion
* `secondary_key` - The Secondary key for the CosmosDB Account.
```

File: website/docs/d/cosmosdb_account.html.markdown [hunk 3]
```diff
@@ -86,13 +86,13 @@ The following attributes are exported:
...
 
-* `primary_readonly_key` - The Primary read-only master Key for the CosmosDB Account.
+* `primary_readonly_key` - The Primary read-only main Key for the CosmosDB Account.
```
Comment (WodansSon) [inline website/docs/d/cosmosdb_account.html.markdown]:
```suggestion
* `primary_readonly_key` - The Primary read-only Key for the CosmosDB Account.
```

File: website/docs/d/cosmosdb_account.html.markdown (hunk 1 above)
Comment (katbyte) [inline website/docs/d/cosmosdb_account.html.markdown:64]:
```suggestion
* `enable_multiple_write_locations` - If multiple write locations are enabled for this Cosmos DB account.
```

File: website/docs/d/cosmosdb_account.html.markdown [hunk 4]
```diff
@@ -86,13 +86,13 @@ The following attributes are exported:
...
 
-* `secondary_readonly_key` - The Secondary read-only master key for the CosmosDB Account.
+* `secondary_readonly_key` - The Secondary read-only main key for the CosmosDB Account.
```
Comment (WodansSon) [inline website/docs/d/cosmosdb_account.html.markdown]:
```suggestion
* `secondary_readonly_key` - The Secondary read-only key for the CosmosDB Account.
```

File: website/docs/d/cosmosdb_account.html.markdown (hunk 2 above)
Comment (katbyte) [inline website/docs/d/cosmosdb_account.html.markdown:91]:
```suggestion
* `secondary_key` - The Secondary key for the CosmosDB Account.
```

File: website/docs/d/cosmosdb_account.html.markdown (hunk 3 above)
Comment (katbyte) [inline website/docs/d/cosmosdb_account.html.markdown:93]:
```suggestion
* `primary_readonly_key` - The Primary read-only Key for the CosmosDB Account.
```

File: website/docs/d/cosmosdb_account.html.markdown (hunk 4 above)
Comment (katbyte) [inline website/docs/d/cosmosdb_account.html.markdown:95]:
```suggestion
* `secondary_readonly_key` - The Secondary read-only key for the CosmosDB Account.
```

File: website/docs/r/cosmosdb_account.html.markdown [hunk 5]
```diff
@@ -104,7 +104,7 @@ The following arguments are supported:
 
 * `virtual_network_rule` - (Optional) Specifies a `virtual_network_rules` resource, used to define which subnets are allowed to access this CosmosDB account.
 
-* `enable_multiple_write_locations` - (Optional) Enable multi-master support for this Cosmos DB account.
+* `enable_multiple_write_locations` - (Optional) Enable multi-main support for this Cosmos DB account.
```
Comment (WodansSon) [inline website/docs/r/cosmosdb_account.html.markdown]:
```suggestion
* `enable_multiple_write_locations` - (Optional) Enable multiple write locations support for this Cosmos DB account.
```

File: website/docs/r/cosmosdb_account.html.markdown [hunk 6]
```diff
@@ -196,13 +196,13 @@ The following attributes are exported:
 
 * `write_endpoints` - A list of write endpoints available for this CosmosDB account.
 
-* `primary_key` - The Primary master key for the CosmosDB Account.
+* `primary_key` - The Primary key for the CosmosDB Account.
 
-* `secondary_key` - The Secondary master key for the CosmosDB Account.
+* `secondary_key` - The Secondary main key for the CosmosDB Account.
```
Comment (WodansSon) [inline website/docs/r/cosmosdb_account.html.markdown]:
```suggestion
* `secondary_key` - The Secondary main for the CosmosDB Account.
```

File: website/docs/r/cosmosdb_account.html.markdown (hunk 5 above)
Comment (katbyte) [inline website/docs/r/cosmosdb_account.html.markdown:107]:
```suggestion
* `enable_multiple_write_locations` - (Optional) Enable multiple write locations for this Cosmos DB account.
```

File: website/docs/r/cosmosdb_account.html.markdown (hunk 6 above)
Comment (katbyte) [inline website/docs/r/cosmosdb_account.html.markdown:201]:
```suggestion
* `secondary_key` - The Secondary key for the CosmosDB Account.
```

File: website/docs/r/kubernetes_cluster.html.markdown [hunk 7]
```diff
@@ -90,7 +90,7 @@ In addition, one of either `identity` or `service_principal` blocks must be spec
 
 * `addon_profile` - (Optional) A `addon_profile` block as defined below.
 
-* `api_server_authorized_ip_ranges` - (Optional) The IP ranges to whitelist for incoming traffic to the masters.
+* `api_server_authorized_ip_ranges` - (Optional) The IP ranges to whitelist for incoming traffic to the main nodes.
```
Comment (WodansSon) [inline website/docs/r/kubernetes_cluster.html.markdown:93]:
```suggestion
* `api_server_authorized_ip_ranges` - (Optional) The IP ranges to allow for incoming traffic to the server nodes.
```

File: website/docs/r/mssql_server.html.markdown [hunk 8]
```diff
@@ -138,7 +138,7 @@ An `extended_auditing_policy` block supports the following:
 
 * `retention_in_days` - (Optional) Specifies the number of days to retain logs for in the storage account.
 
-* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its master database audit events to Azure Monitor.
+* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its main database audit events to Azure Monitor.
```
Comment (WodansSon) [inline website/docs/r/mssql_server.html.markdown:141]:
```suggestion
* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its primary database audit events to Azure Monitor.
```

File: website/docs/r/mssql_server_extended_auditing_policy.html.markdown [hunk 9]
```diff
@@ -64,7 +64,7 @@ The following arguments are supported:
 
 * `storage_account_access_key_is_secondary` - (Optional) Is `storage_account_access_key` value the storage's secondary key?
 
-* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its master database audit events to Azure Monitor.
+* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its main database audit events to Azure Monitor.
```
Comment (WodansSon) [inline website/docs/r/mssql_server_extended_auditing_policy.html.markdown:67]:
```suggestion
* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its primary database audit events to Azure Monitor.
```

File: website/docs/r/sql_server.html.markdown [hunk 10]
```diff
@@ -114,7 +114,7 @@ A `extended_auditing_policy` block supports the following:
 
 * `retention_in_days` - (Optional) Specifies the number of days to retain logs for in the storage account.
 
-* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its master database audit events to Azure Monitor.
+* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its main database audit events to Azure Monitor.
```
Comment (WodansSon) [inline website/docs/r/sql_server.html.markdown:117]:
```suggestion
* `log_monitoring_enabled` - (Optional) Enable audit events to Azure Monitor? To enable server audit events to Azure Monitor, please enable its primary database audit events to Azure Monitor.
```

**Review by WodansSon — CHANGES_REQUESTED** [review]
@katbyte thanks for the PR this mostly LGTM just some minor suggestions 🚀

**Review by WodansSon — APPROVED** [review]
@katbyte thanks for pushing those changes, LGTM! 🚀

**Pull request merged**